docs/user-interface/layouts/horizontalstacklayout.md

```diff
@@ -0,0 +1,166 @@
+
+This example creates a `HorizontalStackLayout` containing a `Rectangle` and a `Label` object, that have ten device-independent units of space between them:
+
+:::image type="content" source="media/horizontalstacklayout/spacing.png" alt-text="HorizontalStackLayout displaying two spaced views screenshot.":::
+
+> [!TIP]
+> The `Spacing` property can be set to negative values to make child views overlap.
+
+## Position and size child views
+
+The size and position of child views within a `HorizontalStackLayout` depends upon the values of the child views' `HeightRequest` and `WidthRequest` properties, and the values of their `VerticalOptions` properties. In a `HorizontalStackLayout`, child views expand to fill the available height when their size isn't explicitly set.
+
+The `VerticalOptions` properties of a `HorizontalStackLayout`, and its child views, can be set to fields from the `LayoutOptions` struct, which encapsulates an *alignment* layout preference. This layout preference determines the position and size of a child view within its parent layout.
+
+> [!TIP]
+> Don't set the `VerticalOptions` property of a `HorizontalStackLayout` unless you need to. The default value of `LayoutOptions.Fill` allows for the best layout optimization. Changing this property has a cost and consumes memory, even when setting it back to its default value.
+
+The following XAML example sets alignment preferences on each child view in the `HorizontalStackLayout`:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="StackLayoutDemos.Views.HorizontalStackLayoutPage">
+    <HorizontalStackLayout Margin="20"
+                           HeightRequest="200">
+        <Label Text="Start"
+               BackgroundColor="Gray"
+               VerticalOptions="Start" />
+        <Label Text="Center"
+               BackgroundColor="Gray"
+               VerticalOptions="Center" />
+        <Label Text="End"
+               BackgroundColor="Gray"
+               VerticalOptions="End" />
+        <Label Text="Fill"
+               BackgroundColor="Gray"
+               VerticalOptions="Fill" />
+    </HorizontalStackLayout>
+</ContentPage>
+```
+
+In this example, alignment preferences are set on the `Label` objects to control their position within the `HorizontalStackLayout`. The `Start`, `Center`, `End`, and `Fill` fields are used to define the alignment of the `Label` objects within the parent `HorizontalStackLayout`:
+
+:::image type="content" source="media/horizontalstacklayout/alignment.png" alt-text="HorizontalStackLayout displaying aligned views screenshot.":::
+
+A `HorizontalStackLayout` only respects the alignment preferences on child views that are in the opposite direction to the orientation of the layout. Therefore, the `Label` child views within the `HorizontalStackLayout` set their `VerticalOptions` properties to one of the alignment fields:
```

Comment:
Maybe "that are in the opposite direction to the orientation of the layout" -> "that are orthogonal to the orientation of the layout"?

Reply:
I agree that orthogonal is better, but experience tells me that (1) a lot of readers won't know what it means, (2) ESL readers won't know what it means, and (3) it won't localise well.